low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -117,20 +114,16 @@ class SSPLBundleGenerate(object):
 
             file_dir = file_name.replace(".tar.gz","")
```

Comment:
`os.path.dirname` method is good rather than replacing some string in the full path.
Also this will improve readability when we refer the dirname later.

Example:
                sspl_log_dir : file_dir+"/sspl/logs/",
                self.sspl_conf_dir : file_dir+"/sspl/config/",

Better be,
                sspl_log_dir :  file_dir + "sspl/logs/",
                self.sspl_conf_dir :  file_dir + "sspl/config/",

Reply:
Done